openapi/components/headers/Link.yaml

```diff
@@ -0,0 +1,3 @@
+schema:
```

Comment:
I used [this](https://github.com/Redocly/api-definitions/blob/master/openapi/paths/3dsecure.yaml) as a reference